### DISCUSSION TITLE
Incorporate grafana graphs into application

### DISCUSSION THREAD

**Discussion by corbosman** [teslamate-org/teslamate#930] [Ideas]
**What would you like to be added**:  

It would be nice if the UI experience could all be inside the teslamate app itself, without having to swap to grafana.  This is possible by loading iframes from teslamate that shows the grafana graphs in a kiosk mode.  Just try it by going to a graph and add:  ```charging-stats?orgId=1&theme=light&kiosk=tv``` . You can even give default timeranges using from=now-1d or whatever. 

**Why is this needed**:  It would feel more like a single app. 

**Additional context**:

**Comment by corbosman** [comment]
So I created my own dashboard instead of teslamate's dashboard, kind of as an example project for a development stack i wanted to play with. It still uses teslamate's grafana dashboards. Just to show how much better it feels when the grafana graphs feel part of the app. Granted, this is not trivial I think as it means you'd have to open up the graphs to the outside world. I am able to whitelist my own clients, but most people probably dont. Maybe some form of proxy could be used instead.

https://www.youtube.com/watch?v=3wFHKfoDWOY

**Comment by denisjoshua** [comment]
Corbosman... I so the tube video and is great :-) 
Can you tell us how can we make same think on owner server please ? 
Thanks a lot. 
Denis

**Comment by gmaslowski** [comment]
@corbosman @denisjoshua 

I've been working on a alternative dashboard to teslamate myself. https://github.com/gmaslowski/teslamate-dash 

So far I've been focusing on adding easy navigation through drives and charges. I'm willing and happy to extend the functionalities.

**Comment by denisjoshua** [comment]
Thanks a lot... but I must find time to test it, cause this period I can't.
